### ISSUE TITLE
Display merge commit changes in changelog

### ISSUE THREAD

**Issue by aburgel** [orhun/git-cliff#85]
**Is your feature request related to a problem? Please describe.**
My team uses a git flow-like workflow where changes are made on feature branches and then merged into main/release branches. We often use merge commits for these merges. Given how git-cliff currently works, the changes included in the merge are not listed as individual changes in the changelog, but instead displayed a single merge commit.

For example, if a change set consists of 2 commits, one of which is a merge commit, even if the merge consists of multiple commits, you'll just see this:

```markdown
**Bug Fixes**
- Fix something

**Other**
- Merge branch 'staging'
```

**Describe the solution you'd like**
I would like the individual commits within the merge commit to be listed as changelog entries, and to exclude the merge commit itself.

**Describe alternatives you've considered**
We thought about changing how we merge branches, like using fast-forwards, but it seems that merge commits are a common enough case that this should be supported.

**Comment by carstencodes** [comment]
Would it be enough to add the following:

[commit.rs#120](https://github.com/orhun/git-cliff/blob/2d721259ca747813c8f9502d84f41b69d5fc685c/git-cliff-core/src/commit.rs#L120)
```rust
    /// commit with two or more parents
    is_merge_commit: bool,
```

[commit.rs#151](https://github.com/orhun/git-cliff/blob/2d721259ca747813c8f9502d84f41b69d5fc685c/git-cliff-core/src/commit.rs#L151)
```rust
    is_merge_commit: commit.parent_count > 1,
```

This way, you could be able to filter in your template?

**Comment by aburgel** [comment]
The issue I was seeing was that the commits from the second parent of the merge were not shown. You would only see the merge commit itself, not what it "contained".

Your solution would be helpful to filter out the merge commit itself (which is not the helpful to see in changelog), but that still leaves out the merged-in commits.

**Comment by orhun** [comment]
Hello! 🐻 

Sorry for getting back to this so late - I was too caught up in the work life and working on a project at this scale was a bit overwhelming. But now I'm back on track and lots of exciting stuff are on the way!

Anyhoo, I implemented the suggested change in #389 - feel free to test it out!

> The issue I was seeing was that the commits from the second parent of the merge were not shown. You would only see the merge commit itself, not what it "contained".

@aburgel I tried to get the full list of "merged" commits from a parent but that did not seem possible with `git2`. I searched for examples but couldn't find anything :/
